src/includes/performance-monitoring/configuration/javascript.mdx

```diff
@@ -1,41 +1,42 @@
-
-To get started with performance monitoring using Sentry's JavaScript SDK, first install the `@sentry/browser` and `@sentry/apm` packages:
+To get started with performance monitoring using Sentry's JavaScript SDK, first install the `@sentry/browser` and `@sentry/tracing` packages:
 
 ```bash
 # Using yarn
-$ yarn add @sentry/browser @sentry/apm
+$ yarn add @sentry/browser @sentry/tracing
 
 # Using npm
-$ npm install @sentry/browser @sentry/apm
+$ npm install @sentry/browser @sentry/tracing
 ```
 
+<Alert level="warning" title="Note"><markdown>
+
+We switched our package name from `@sentry/apm` to `@sentry/tracing`. If you were previously using `@sentry/apm`, please see [Switching to the tracing package](/product/performance/apm-to-tracing/)
+
+</markdown></Alert>
+
 Next, initialize the integration in your call to `Sentry.init`:
 
 ```jsx
-import * as Sentry from '@sentry/browser';
-import { Integrations as ApmIntegrations } from '@sentry/apm';
+import * as Sentry from "@sentry/browser";
+import { Integrations } from "@sentry/tracing";
```

Comment:
Did you change it to just `Integrations` for simplicity? It had been `ApmIntegrations` in order that people be able to import the integrations package as `Integrations` also. You might consider `TracingIntegrations`.

Reply:
could we just improve the exports here?

```
import { Tracing } from "..."
```

What else would this even expose? Why do we need it to be contained in Integrations?

Reply:
> could we just improve the exports here?

I eventually came to the same idea as I read on, so I'm upgrading my "we could" below to a "I think we should." (Provided it doesn't break anything, which I don't believe it would.)

Reply:
`import { Integrations } from "@sentry/tracing";`

`Integrations` is an object containing `{ Express, BrowserTracing }`

We could separate this yes, I just chose to make the behaviour consistent with `@sentry/apm`. It wouldn't break anything to change the imports, we would just have to update the upgrade process.

Reply:
After revising this, I decided to not change the imports. In the CDN bundle, we keep everything together in `Sentry.Integrations`, hence why we chose to keep it the same way in the npm package. I think we should for now keep the same behaviour in `@sentry/apm` and we can revisit integrations in general in v6.